client/components/sites-popover/index.jsx

```diff
@@ -8,7 +8,6 @@ var React = require( 'react' ),
  * Internal dependencies
  */
 var Popover = require( 'components/popover' ),
-	sitesList = require( 'lib/sites-list' )(),
```

Comment:
>I think this was here to make the component self-sufficient in a way. cc @aduth

I honestly don't recall, though I'd agree that could be an argument for keeping it the way it was. Was there a reason this change was made?